datalad/distribution/get.py

```diff
@@ -503,53 +509,83 @@ def _install_necessary_subdatasets(
 
 
 def _recursive_install_subds_underneath(ds, recursion_limit, reckless, start=None,
-                                        refds_path=None, description=None):
+                 refds_path=None, description=None, jobs=None, producer_only=False):
     if isinstance(recursion_limit, int) and recursion_limit <= 0:
         return
+    if jobs == "auto":
+        # be safe -- there might be ssh logins etc. So no parallel
+        jobs = 0
```

Comment:
hm, re-reviewing my code stumbled upon this .  I guess should be removed since I think it came from the time when I had "auto" by default making 5 jobs.  ATM "auto" should be a single job, which should be ok for install.  Or if not to remove - just making a mental note that I saw it ;)